reo/src/techs.py

```diff
@@ -146,7 +148,10 @@ def __init__(self, dfm, degradation_pct, time_steps_per_hour=1, acres_per_kw=6e-
             else:  # All other tilts come from lookup table included in the array_type_to_tilt_angle dictionary above
                 self.tilt = PV.array_type_to_tilt_angle[kwargs.get('array_type')]
 
-        self.pvwatts = PVWatts(time_steps_per_hour=self.time_steps_per_hour, azimuth=self.azimuth, tilt=self.tilt, **self.kwargs)
+        if self.prod_factor_series_kw is not None:  # then don't call PVWatts
+            self.station = (kwargs.get("latitude", 0), kwargs.get("longitude", 0), 0)
```

Comment:
Do we need to set station coords? Lat and long are required so we shouldn't get (0,0), but that is is a real point and may be confusing if it comes back from the API. Is there a problem with (None, None)?

Reply:
I was trying to preserve old behavior, but we don't need to set them. (Old behavior was to call PVWatts to get station lat/lon even when user provider their own prod_factor).

Reply:
@NLaws thanks - I'll approve this and check with Andrew about the 0,0 issue and decide how to handle that after that